src/cbmc/bmc.cpp

```diff
@@ -76,10 +76,10 @@ void bmct::error_trace()
       json_arrayt &result_array=json_result["results"].make_array();
       json_objectt &result=result_array.push_back().make_object();
       const goto_trace_stept &step=goto_trace.steps.back();
-      result["property"]=
-        json_stringt(id2string(step.pc->source_location.get_property_id()));
-      result["description"]=
-        json_stringt(id2string(step.pc->source_location.get_comment()));
+      result["property"]=json_stringt(
```

Comment:
Violates "Break after `=` if it is part of an assignment" of the coding standard. Applies elsewhere in this PR as well.